File: manager/src/main/java/com/mydemo/manager/service/ContentService.java
package com.mydemo.manager.service;

import com.mydemo.manager.entity.Content;

import java.util.List;

public interface ContentService {
    public List<Content> findAll();
    public Content findOne(int id);
    public int dele(Integer[] id);
    public int update(Content content);
    public int add(Content content);
    public int setStatus(Integer[] ids,String status);
}

File: manager/src/main/resources/templates/js/service/contentService.js
app.service('contentService',function($http){

    this.findAll=function () {
        return $http.get('../content/findAll');
    }
    this.findOne=function (id) {
        return $http.get('../content/findOne?id='+id);
    }
    this.update=function (entity) {
        return $http.post('../content/update',entity);
    }
    this.add=function (entity) {
        return $http.post('../content/add',entity);
    }
    this.dele=function (ids) {
        return $http.get('../content/dele?ids='+ids);
    }
    this.setStatus=function (ids,status) {
        return $http.get('../content/setStatus?ids='+ids+'&status='+status)
    }
});